jfr-daemon/build.gradle.kts

```diff
@@ -15,26 +16,20 @@ java {
     disableAutoTargetJvm()
 }
 
-dependencies {
-    api(project(":jfr-mappers"))
-    api("org.slf4j:slf4j-api:${Versions.slf4j}")
-    api("org.apache.logging.log4j:log4j-slf4j-impl:${Versions.log4j}")
-    api("org.apache.logging.log4j:log4j-core:${Versions.log4j}")
-    api("com.newrelic.telemetry:telemetry-http-java11:${Versions.newRelicTelemetry}")
-    implementation("com.google.code.gson:gson:${Versions.gson}")
-}
-
-tasks.shadowJar {
-    archiveClassifier.set("")
-    manifest {
-        attributes(
-                "Main-Class" to "com.newrelic.jfr.daemon.JFRDaemon",
-                "Implementation-Version" to project.version
-        )
+jpmsExtraModules {
+    module("mockito-junit-jupiter-${mockitoVersion}.jar", "mockito.junit.jupiter", mockitoVersion) {
+        exports("org.mockito.junit.jupiter")
+    }
+    module("objenesis-${objenesisVersion}.jar", "org.objenesis", objenesisVersion) {
+        exports("org.objenesis")
     }
 }
 
-tasks.named("build") { dependsOn("shadowJar") }
+dependencies {
+    implementation(project(":jfr-mappers"))
+    implementation(jpms.asModule("org.apache.logging.log4j:log4j-core:${log4jVersion}", "org.apache.logging.log4j.core"))
```

Comment:
I've tried to create and run the jlink artifacts, and I see a variety of thinkgs in the `jfr-daemon/build/jlink` directory, but when I run them its not clear whether they're working or not. I think there may be a problem with the logging. 

I notice that the slf4j dependencies are missing and wondering if that has something to do with it. What's the expected behavior of logging with jlink artifacts?